mock-infrastructure/src/main/java/com/sequenceiq/mock/verification/intercept/EndpointTestDecoratorInterceptor.java

```diff
@@ -73,10 +75,21 @@ private Map<String, String> getRequestParams(HttpServletRequest request) {
             });
             return params;
         }
+        if (request instanceof StandardMultipartHttpServletRequest) {
+            StandardMultipartHttpServletRequest multipartHttpServletRequest = (StandardMultipartHttpServletRequest) request;
+            multipartHttpServletRequest.getMultiFileMap().getFirst("file");
```

Comment:
Is it a leftover code line? No return value is used here.

Reply:
Yes, it is a leftover code, I missed it :| thank you! :)